doc/testing.md

```diff
@@ -0,0 +1,17 @@
+## Continuous integration and testing for OpenQuake
+
+### Automatic testing
+
+The OpenQuake code is automatically tested by a [Continuous integration](https://ci.openquake.org) system at every merge and every night. This system uses [Jenkins](http://jenkins-ci.org/).
+
+Current build status is  [![Build Status](https://ci.openquake.org/job/master_oq-hazardlib/badge/icon)](https://ci.openquake.org/job/master_oq-hazardlib/)
+
+### Manual testing
+
+The full suite of tests for the OpenQuake Hazardlib can be run using `nose`
+
+```
+nosetests -v -a '!slow'
+```
```

Comment:
With -a'!slow' we skip some tests, so it is not the full suite. So I would write `The suite of tests for` without the word `full`